server/apps/client.go

```diff
@@ -101,6 +100,7 @@ func (c *client) post(toApp *api.App, fromMattermostUserID string, url string, m
 		return nil, err
 	}
 	req.Header.Set(OutgoingAuthHeader, "Bearer "+jwtoken)
+	req.Header.Set("Content-Type", "application/json")
```

Comment:
@levb, I added this commit to the PR.  This is needed for clients receiving the post with `json` data structures